From 56a54641c6d06a1bd3aac4128320cb3fc2dc5d82 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 18 Nov 2014 13:54:08 -0500
Subject: [redhat] kabi: Adding symbol __dynamic_dev_dbg (lib/dynamic_debug.c) DYNAMIC DEBUG

Message-id: <18d87149cc86c581d8be87255f8bbc8b2fa42f21.1416318767.git.poros@redhat.com>
Patchwork-id: 100125
O-Subject: [PATCH RHEL7.1 03/63] kabi: Adding symbol __dynamic_dev_dbg (lib/dynamic_debug.c) DYNAMIC DEBUG
Bugzilla: 1070274
RH-Acked-by: Don Zickus <dzickus@redhat.com>

Bugzilla: 1070274
    https://bugzilla.redhat.com/show_bug.cgi?id=1070274

    Description:
    ============
    Used by the dynamic_dev_dbg() macro.
    We want to use dynamic_dev_dbg().
    Symbol used in drbd

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7.0 08/80] kabi: Adding symbol __dynamic_dev_dbg (lib/dynamic_debug.c)
 Jiri Olsa <jolsa@redhat.com> (Tue,  4 Mar 2014 16:31:08 +0100)
 Description:
 ============
 Used by the dynamic_dev_dbg() macro.
 We want to use dynamic_dev_dbg().

 Reviews:
 ========
 Re: [PATCH RHEL7.0 08/80] kabi: Adding symbol __dynamic_dev_dbg (lib/dynamic_debug.c)
 Prarit Bhargava <prarit@redhat.com> (Wed, 05 Mar 2014 07:53:53 -0500)
 Acked-by: Prarit Bhargava <prarit@redhat.com>
 P.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__dynamic_dev_dbg
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__dynamic_dev_dbg
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__dynamic_dev_dbg

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__dynamic_dev_dbg b/redhat/kabi/kabi-module/kabi_ppc64/__dynamic_dev_dbg
new file mode 100644
index 0000000..f5c7ba8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__dynamic_dev_dbg
@@ -0,0 +1,2 @@
+#1-
+0xec6fc505 __dynamic_dev_dbg vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__dynamic_dev_dbg b/redhat/kabi/kabi-module/kabi_s390x/__dynamic_dev_dbg
new file mode 100644
index 0000000..84e6976
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__dynamic_dev_dbg
@@ -0,0 +1,2 @@
+#1-
+0x756b2b5e __dynamic_dev_dbg vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__dynamic_dev_dbg b/redhat/kabi/kabi-module/kabi_x86_64/__dynamic_dev_dbg
new file mode 100644
index 0000000..80b19c7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__dynamic_dev_dbg
@@ -0,0 +1,2 @@
+#1-
+0xc140ad72 __dynamic_dev_dbg vmlinux EXPORT_SYMBOL
-- 
1.7.1